project.clj

```diff
@@ -85,6 +85,8 @@
                  [puppetlabs/comidi]
                  [puppetlabs/i18n]
 
+                 [com.fasterxml.jackson.module/jackson-module-afterburner "2.7.1"]
```

Comment:
Does this version have to be synchronized now with the version of jackson transitively pulled in by cheshire from clj-parent?

Reply:
Not sure. @justinstoller ?

Reply:
ppfft, this is only an issue because of https://tickets.puppetlabs.com/browse/SERVER-1731 jrjackson vendors the jars it needs (and is set to 2.9.4 currently).

It synchronizes its versions of jackson projects and we probably should too. That's probably a good enough reason put it into clj-parent?

Reply:
I talked with @Magisus and she said she'll move it into clj-parent